.github/workflows/npm-release.yml

```
@@ -0,0 +1,94 @@
+  release:
+    name: Release
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write # to be able to publish a GitHub release
+      issues: write # to be able to comment on released issues
+      pull-requests: write # to be able to comment on released pull requests
+      id-token: write # to enable use of OIDC for npm provenance
+    outputs:
+      ARTIFACT_NAME: ${{ steps.node.outputs.artifact-name }}
+      VERSION: ${{steps.node-release.outputs.VERSION}}
+    env: 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)